ui/app/js/controllers/rule_edit_controller.js

```diff
@@ -59,6 +73,7 @@ function ($scope, $modalInstance, CSRF, Rules, Releases, rule, pr_ch_options) {
       .error(function(response) {
         if (typeof response === 'object') {
           $scope.errors = response;
+          Helpers.addErrorFields($scope.errors);
```

Comment:
I'm sorry if I sent you down the wrong path here...but I don't think it's a good to change the overall error handling. It's tightly wound with the response that the backend returns, and I'm concerned we're going to stop showing some errors by hardcoding specific fields. For example: the backend has many cases where "detail" or "exception" will be in the returned error, and it looks like you're removing support for showing them.

Reply:
I think so too. That being said, I could leave the backend error handling logic and display and for now, go with an additional variable for UI validation errors and display and maybe raise a separate ticket for harmonization of the two?

Reply:
That seems like a good plan.